atom/index.js

```diff
@@ -1,81 +1,93 @@
 import { clean } from '../clean-stores/index.js'
-
-let listenerQueue = []
+import { getStoreState, globalContext } from '../context/index.js'
 
 export let atom = (initialValue, level) => {
-  let listeners = []
   let $atom = {
+    ctx: globalContext,
     get() {
-      if (!$atom.lc) {
-        $atom.listen(() => {})()
+      let state = getStoreState(this, $atom)
+      if (!state.lc) {
+        this.listen(() => {})()
       }
-      return $atom.value
+      return state.v
     },
+    iv: initialValue,
     l: level || 0,
-    lc: 0,
-    listen(listener, listenerLevel) {
-      $atom.lc = listeners.push(listener, listenerLevel || $atom.l) / 2
+    get lc() {
```

Comment:
Nice 😈

Reply:
That's Bogdan's idea! Yeah, it's good.
cc @TrySound